composer.json

```
@@ -20,7 +20,7 @@
     "pimcore/personalization-bundle": "^1.0",
     "pimcore/google-marketing-bundle": "^1.0",
     "knplabs/knp-paginator-bundle": "^6.0.0",
-    "symfony/webpack-encore-bundle": "^1.13.2",
+    "symfony/webpack-encore-bundle": "^1.17",
```

@mattamon I think we could support both versions. Also in admin-ui-classic-bundle. WDYT?
```suggestion
    "symfony/webpack-encore-bundle": "^1.13.2 || ^2.0",
```

---

yeah but I would at least already set it to 1.17 so like ^1.17 || ^2.0